backend/heroes.py

```diff
@@ -210,12 +254,14 @@ class CrocodileMan(Hero):
   name = 'CrocodileMan'
   title = 'Smelly Reptile'
   abilities = []
-  action_classes = [FlipWeekest, PushBackAttack]
+  action_classes = [FlipWeakest, PushBackAttack]
   shape = shapes.krokotyuk
-  def before_step(self):
-    normal_influance = self.influence_base + self.level * self.influence_per_level
-    self.influence = normal_influance * (
-      (self.max_loyalty - abs(self.loyalty)) / self.max_loyalty + 1)
+  anger = 0
+  def hit(self, amount):
+    super().hit(amount)
+    self.anger += amount
+    normal_influence = self.influence_base + self.level * self.influence_per_level
+    self.influence = normal_influence * (self.anger / self.max_loyalty + 1)
```

Comment:
I changed it so that he doesn't get weaker from healing.

Reply:
Looks good!